CHANGELOG.md

```diff
@@ -15,8 +15,8 @@
 - Added support for `by`, `left_by`, and `right_by` for `pd.merge_asof`.
 
 #### Bug Fixes
-
```

Comment:
@sfc-gh-mvashishtha Can we keep this line?

Reply:
Done.